spec/factories/providers.rb

```diff
@@ -29,6 +29,7 @@
     address3 { Faker::Address.city }
     address4 { Faker::Address.state }
     postcode { Faker::Address.postcode }
+    scitt 'N'
```

Comment:
```suggestion
    scitt { 'N' }
```
otherwise I'm not sure you can override this?

Reply:
Apologies, I think it was probably OK before, it needs the block if it's dynamic which it isn't in this case. Anyhoo the block doesn't hurt.